docs/source/installation.rst

```diff
@@ -3,43 +3,119 @@
+This will install ``sktime`` with core dependencies, excluding soft dependencies.
 
-Retrieving other stable versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Currently, there is no easy route to install ``sktime`` with maximum dependencies via ``conda``.
 
-If you want to build a previous version, you can run:
+Community contributions towards this, e.g., via conda metapackages, would be appreciated.
+
+
+Release versions - troubleshooting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Module not found
+""""""""""""""""
+
+The most frequent reason for "module not found" errors is installing ``sktime`` with minimum dependencies
+and using an estimator which interfaces a packages that has not been installed in the environment.
+To resolve this, install the missing package, or install ``sktime`` with maximum dependencies (see above).
+
+
+Facebook prophet
+""""""""""""""""
+
+A frequent issue arises with installation of facebook prophet when installing via ``pip``, especially on Windows systems.
+
+Potential workaround no.1, install manually via ``conda-forge``:
+
+.. code-block:: bash
+
+    conda install -c conda-forge pystan
+    conda install -c conda-forge prophet
+
+The remaining packages can be installed via ``pip`` or ``conda``.
+
+Potential workaround no.2, install ``pystan`` with ``no-cache`` parameter:
+
+.. code-block:: bash
+
+    pip install pystan --no-cache
+
+Potential workaround no.3, if on Windows: use WSL (Windows Subsystem for Linux), see end of section `Windows 8.1 and higher`_.
+
+
+numpy or C related issues
+"""""""""""""""""""""""""
+
+``numpy`` and C related errors on Windows based systems are potentially resolved by installing Build Tools for Visual Studio 2019 or 2017.
+
+
+Development versions
+--------------------
+To install the the latest development version of ``sktime``, or earlier versions, the sequence of steps is as follows:
+
+Step 1 - ``git`` clone the ``sktime`` repository, the latest version or an earlier version.
+Step 2 - ensure build requirements are satisfied
+Step 3 - ``pip`` install the package from a ``git`` clone, with the ``editable`` parameter.
+
+Detail instructions for all steps are given below.
+For brevity, we discuss steps 1 and 3 first; step 2 is discussed at the end, as it will depend on the operating system.
+
+Step 1 - git cloning the repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``sktime`` repository should be cloned to a local directory, using a graphical user interface, or the command line.
+
+Using the ``git`` command line, the sequence of commands to install the latest version is as follows:
+
+1. Clone the repository: :code:`git clone https://github.com/alan-turing-institute/sktime.git`
+2. Move into the root directory of the local clone: :code:`cd sktime`
+3. Make sure you are on the main branch: :code:`git checkout main`
+4. Make sure your local version is up-to-date: :code:`git pull`
+
+To build a previous version, replace line 3 with:
```

Comment:
in the code block below this line we need to remove the "``" marks

Reply:
done